Docker - Environment Variables not working

With the latest release the environment variables passed in to the container no longer seem to be being used. In this case the poller is trying to login to `127.0.0.1` despite `UP_UNIFI_URL` being present.

`docker inspect unifi-poller`
```
            "Env": [
                ...
                "UP_UNIFI_URL=https://unifi.jamiewood.io",
                "UP_POLLING_INTERVAL=1m",
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
            ],
```

`docker logs unifi-poller`
```
2019/12/02 10:12:03 [INFO] UniFi Poller v1.6.0-553 Starting Up! PID: 1
2019/12/02 10:12:03 [ERROR] unifi controller: Post https://127.0.0.1:8443/api/login: dial tcp 127.0.0.1:8443: connect: connection refused
```

--- a/pkg/poller/config.go
+++ b/pkg/poller/config.go
@@ -114,8 +114,9 @@ func (c *Config) ParseENV() error {
 	t := reflect.TypeOf(Config{}) // Get tag names from the Config struct.
 	// Loop each Config struct member; get reflect tag & env var value; update config.
 	for i := 0; i < t.NumField(); i++ {
-		tag := strings.ToUpper(t.Field(i).Tag.Get("json")) // Get the ENV variable name from capitalized "json" struct tag
-		env := os.Getenv(ENVConfigPrefix + tag)            // Then pull value from OS.
+		tag := t.Field(i).Tag.Get("json")                 // Get the ENV variable name from "json" struct tag
+		tag = strings.Split(strings.ToUpper(tag), ",")[0] // Capitalize and remove ,omitempty suffix
+		env := os.Getenv(ENVConfigPrefix + tag)           // Then pull value from OS.
 		if tag == "" || env == "" {
 			continue // Skip if either are empty.
 		}

